2019-09-06 18:31:17,621--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:31:18,780--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:31:19,067--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:31:19,068--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:31:19,265--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:31:19,266--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:31:19,457--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:31:19,458--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:31:19,619--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:31:19,620--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:31:19,791--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:31:19,794--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:33:32,145--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:33:33,298--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:33:33,942--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:33:33,943--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:33:34,217--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:33:34,218--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:33:34,407--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:33:34,407--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:33:34,586--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:33:34,587--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:33:34,786--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:33:34,788--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:35:16,965--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:35:18,073--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:35:18,298--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:35:18,298--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:35:18,510--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:35:18,511--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:35:18,747--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:35:18,747--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:35:18,959--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:35:18,959--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:35:19,123--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:35:19,125--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:37:06,946--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:37:08,075--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:37:08,330--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:37:08,330--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:37:08,524--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:37:08,525--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:37:08,767--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:37:08,768--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:37:08,970--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:37:08,971--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:37:09,165--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:37:09,167--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:38:55,373--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:38:56,664--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:38:56,904--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:38:56,904--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:38:57,102--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:38:57,103--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:38:57,284--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:38:57,284--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:38:57,457--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:38:57,458--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:38:57,632--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:38:57,635--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:41:35,180--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:41:36,535--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:41:36,748--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:41:36,749--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:41:36,972--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:41:36,973--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:41:37,155--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:41:37,156--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:41:37,314--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:41:37,315--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:41:37,494--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:41:37,522--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:42:16,536--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:42:17,690--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:17,905--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:42:17,906--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:18,099--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:42:18,100--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:18,280--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:42:18,281--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:18,456--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:42:18,457--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:18,647--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:42:18,653--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 18:42:54,294--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 18:42:55,608--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:55,819--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 18:42:55,820--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:56,033--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 18:42:56,034--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:56,242--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 18:42:56,243--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:56,434--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 18:42:56,435--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 18:42:56,638--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 18:42:56,638--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:56,882--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:注册成功测试通过
2019-09-06 18:42:56,882--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:57,105--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-06 18:42:57,106--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:57,270--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-06 18:42:57,270--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:57,438--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-06 18:42:57,438--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:57,604--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-06 18:42:57,604--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:57,821--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-06 18:42:57,821--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 18:42:57,991--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-06 18:42:57,993--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-06 19:00:47,398--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-06 19:00:48,623--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 19:00:49,037--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-06 19:00:49,037--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 19:00:49,294--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-06 19:00:49,295--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 19:00:49,540--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-06 19:00:49,541--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 19:00:49,705--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-06 19:00:49,706--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-06 19:00:49,863--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-06 19:00:49,864--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:50,097--[test_register.py--line:44]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-06 19:00:50,099--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:50,293--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-06 19:00:50,294--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:50,463--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-06 19:00:50,464--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:50,626--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-06 19:00:50,626--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:50,831--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-06 19:00:50,832--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:51,005--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-06 19:00:51,006--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-06 19:00:51,179--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-06 19:00:51,182--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 10:57:23,202--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 10:57:25,087--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 10:57:25,806--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 10:57:25,807--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 10:57:25,982--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 10:57:25,983--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 10:57:26,196--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 10:57:26,198--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 10:57:26,403--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 10:57:26,404--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 10:57:26,651--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 10:57:26,652--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:26,844--[test_register.py--line:44]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 10:57:27,026--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:27,224--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-09 10:57:27,225--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:27,385--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-09 10:57:27,385--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:27,591--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-09 10:57:27,592--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:27,769--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-09 10:57:27,769--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:27,968--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-09 10:57:27,968--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 10:57:28,126--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-09 10:57:28,128--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 11:09:01,469--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 11:09:02,757--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:09:03,335--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 11:09:03,336--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:09:03,517--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 11:09:03,517--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:09:03,703--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 11:09:03,704--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:09:03,981--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 11:09:03,982--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:09:04,241--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 11:09:04,242--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:04,443--[test_register.py--line:44]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 11:09:04,445--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:04,682--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-09 11:09:04,683--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:04,839--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-09 11:09:04,839--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:05,012--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-09 11:09:05,013--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:05,201--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-09 11:09:05,202--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:05,372--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-09 11:09:05,373--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:09:05,564--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-09 11:09:05,566--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 11:11:24,528--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 11:11:25,669--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:11:25,876--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 11:11:25,876--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:11:26,053--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 11:11:26,054--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:11:26,240--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 11:11:26,240--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:11:26,407--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 11:11:26,408--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 11:11:26,566--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 11:11:26,566--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:26,747--[test_register.py--line:44]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 11:11:26,748--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:26,963--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-09 11:11:26,963--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:27,177--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-09 11:11:27,178--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:27,331--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-09 11:11:27,331--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:27,512--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-09 11:11:27,512--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:27,663--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-09 11:11:27,664--[test_register.py--line:33]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 11:11:27,837--[test_register.py--line:51]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-09 11:11:27,840--[suite_run.py--line:38]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 14:00:19,207--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 14:00:20,515--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:00:21,138--[test_login.py--line:55]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 14:00:21,138--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:00:21,380--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 14:00:21,381--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:00:21,613--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 14:00:21,613--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:00:21,777--[test_login.py--line:55]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 14:00:21,778--[test_login.py--line:35]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:00:21,958--[test_login.py--line:55]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 14:00:21,959--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:22,164--[test_register.py--line:43]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 14:00:22,218--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:22,422--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-09 14:00:22,424--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:22,599--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-09 14:00:22,600--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:22,782--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-09 14:00:22,782--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:22,994--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-09 14:00:22,995--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:23,176--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-09 14:00:23,177--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:00:23,345--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-09 14:00:23,347--[suite_run.py--line:39]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 14:03:52,659--[suite_run.py--line:17]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 14:03:54,131--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:03:54,428--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 14:03:54,429--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:03:54,732--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 14:03:54,732--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:03:54,985--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 14:03:54,987--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:03:55,187--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 14:03:55,187--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:03:55,428--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 14:03:55,429--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:55,692--[test_register.py--line:43]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 14:03:55,695--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:55,926--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-09 14:03:55,926--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:56,102--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-09 14:03:56,102--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:56,273--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-09 14:03:56,274--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:56,462--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-09 14:03:56,463--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:56,639--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-09 14:03:56,640--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:03:56,813--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-09 14:03:56,819--[suite_run.py--line:39]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 14:10:01,836--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 14:10:02,972--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:10:03,212--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 14:10:03,213--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:10:03,398--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 14:10:03,399--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:10:03,603--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 14:10:03,603--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:10:03,767--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 14:10:03,767--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 14:10:03,937--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 14:10:03,937--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:04,135--[test_register.py--line:43]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 14:10:04,137--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:04,327--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机号已注册测试通过
2019-09-09 14:10:04,327--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:04,494--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码长度小于6测试通过
2019-09-09 14:10:04,494--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:04,665--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码长度大于18测试通过
2019-09-09 14:10:04,667--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:04,838--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机号为空测试通过
2019-09-09 14:10:04,839--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:05,014--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:密码为空测试通过
2019-09-09 14:10:05,015--[test_register.py--line:32]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 14:10:05,174--[test_register.py--line:50]--日志等级:DEBUG--具体内容：用例:手机格式错误测试通过
2019-09-09 14:10:05,176--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 15:44:29,866--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 15:44:33,637--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:44:33,955--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 15:44:33,956--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:44:34,144--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 15:44:34,145--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:44:34,402--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 15:44:34,403--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:44:34,589--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 15:44:34,590--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:44:34,744--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 15:44:34,745--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 15:45:37,325--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 15:45:43,679--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:45:43,910--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 15:45:43,910--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:45:44,146--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 15:45:44,147--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:45:44,342--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 15:45:44,342--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:45:44,509--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 15:45:44,509--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:45:44,688--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 15:45:44,689--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:44,837--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:44,982--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:45,072--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:45,139--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:45,216--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:45,296--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:45:45,381--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 15:46:59,571--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 15:47:00,944--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:47:01,165--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 15:47:01,166--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:47:01,365--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 15:47:01,365--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:47:01,601--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 15:47:01,603--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:47:01,806--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 15:47:01,807--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 15:47:01,993--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 15:47:01,994--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,093--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,186--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,240--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,314--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,372--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,435--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 15:47:02,495--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 16:04:34,161--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 16:04:38,012--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:04:38,229--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 16:04:38,230--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:04:38,417--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 16:04:38,418--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:04:38,621--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 16:04:38,622--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:04:38,784--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 16:04:38,785--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:04:38,951--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 16:04:38,952--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,048--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,124--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,181--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,239--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,296--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,350--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:04:39,411--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 16:08:35,191--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 16:08:37,636--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:08:37,886--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 16:08:37,886--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:08:38,131--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 16:08:38,131--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:08:38,371--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 16:08:38,371--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:08:38,530--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 16:08:38,531--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:08:38,671--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 16:08:38,672--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:38,754--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:38,841--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:38,902--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:38,957--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:39,024--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:39,076--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:08:39,150--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 16:14:36,623--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 16:14:38,125--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:14:38,412--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:14:38,545--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:14:38,708--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:14:38,809--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:14:38,912--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,138--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,233--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,295--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,351--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,410--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,474--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:14:39,539--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 16:15:27,559--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 16:15:29,060--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:15:29,257--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 16:15:29,257--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:15:29,567--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 16:15:29,567--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:15:30,759--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 16:15:30,760--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:15:30,928--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 16:15:30,929--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:15:31,076--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 16:15:31,076--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,178--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,267--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,323--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,385--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,443--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,519--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:15:31,585--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 16:20:40,432--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 16:20:42,807--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:20:42,999--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 16:20:43,000--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:20:43,172--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 16:20:43,174--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:20:43,366--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 16:20:43,366--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:20:43,520--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 16:20:43,520--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:20:43,679--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 16:20:43,680--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:43,762--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:43,841--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:43,899--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:43,951--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:44,013--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:44,068--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:20:44,127--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 16:21:27,768--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 16:21:31,937--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:21:32,218--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 16:21:32,220--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:21:32,434--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 16:21:32,435--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:21:32,601--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 16:21:32,601--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:21:32,753--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 16:21:32,754--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 16:21:32,901--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 16:21:32,901--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:32,979--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:33,059--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:33,120--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:33,177--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:33,237--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:33,295--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 16:21:33,363--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 17:47:32,257--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 17:47:42,667--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:47:43,005--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 17:47:43,006--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:47:43,352--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 17:47:43,353--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:47:43,652--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 17:47:43,653--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:47:43,904--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 17:47:43,904--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:47:44,178--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 17:47:44,182--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 17:54:56,673--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 17:54:58,346--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:54:58,600--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 17:54:58,601--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:54:58,874--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 17:54:58,875--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:54:59,204--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 17:54:59,205--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:54:59,442--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 17:54:59,442--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:54:59,714--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 17:54:59,816--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:54:59,975--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:55:00,066--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:55:00,137--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:55:00,239--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:55:00,313--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:55:00,396--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:55:00,477--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 17:56:15,608--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 17:56:17,916--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:56:18,132--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 17:56:18,132--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:56:18,382--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 17:56:18,382--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:56:18,618--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 17:56:18,619--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:56:18,780--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 17:56:18,780--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:56:18,959--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 17:56:19,203--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 17:56:19,638--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 17:56:19,808--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 17:56:20,009--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 17:56:20,176--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 17:56:20,351--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 17:56:20,352--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,445--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,534--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,612--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,681--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,739--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,838--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:56:20,911--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 17:57:53,216--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 17:57:55,528--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:57:55,811--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 17:57:55,812--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:57:56,035--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 17:57:56,035--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:57:56,272--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 17:57:56,273--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:57:56,512--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 17:57:56,512--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:57:56,795--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 17:57:57,004--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 17:57:57,421--[test_recharge.py--line:52]--日志等级:INFO--具体内容：用例充值成功执行未通过
2019-09-09 17:57:57,421--[test_recharge.py--line:53]--日志等级:ERROR--具体内容：2000 != Decimal('0.00')
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_recharge.py", line 49, in test_recharge
    self.assertEqual(money,change_money)
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 2000 != Decimal('0.00')
2019-09-09 17:57:57,631--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 17:57:57,820--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 17:57:57,997--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 17:57:58,187--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 17:57:58,371--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 17:57:58,372--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,469--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,566--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,629--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,683--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,745--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,911--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:57:58,989--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 17:58:55,200--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 17:58:56,661--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:58:56,899--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 17:58:56,900--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:58:57,126--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 17:58:57,127--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:58:57,384--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 17:58:57,385--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:58:57,599--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 17:58:57,601--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 17:58:57,807--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 17:58:58,005--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 17:58:58,431--[test_recharge.py--line:52]--日志等级:INFO--具体内容：用例充值成功执行未通过
2019-09-09 17:58:58,431--[test_recharge.py--line:53]--日志等级:ERROR--具体内容：2000 != Decimal('0.00')
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_recharge.py", line 49, in test_recharge
    self.assertEqual(money,change_money)
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 2000 != Decimal('0.00')
2019-09-09 17:58:58,678--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 17:58:58,848--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 17:58:59,013--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 17:58:59,183--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 17:58:59,362--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 17:58:59,362--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,457--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,550--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,607--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,685--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,746--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,824--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 17:58:59,913--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:04:20,763--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:04:23,231--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:04:23,455--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:04:23,456--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:04:23,669--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:04:23,670--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:04:23,899--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:04:23,900--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:04:24,117--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:04:24,118--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:04:24,341--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:04:24,548--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:04:25,006--[test_recharge.py--line:52]--日志等级:INFO--具体内容：用例充值成功执行未通过
2019-09-09 18:04:25,006--[test_recharge.py--line:53]--日志等级:ERROR--具体内容：2000 != Decimal('0.00')
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_recharge.py", line 49, in test_recharge
    self.assertEqual(money,change_money)
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 2000 != Decimal('0.00')
2019-09-09 18:04:25,175--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:04:25,357--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:04:25,548--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:04:25,725--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:04:25,899--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:04:25,900--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,008--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,131--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,195--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,256--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,336--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,400--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:04:26,467--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:15:06,982--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:15:12,980--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:15:13,217--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:15:13,217--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:15:13,461--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:15:13,462--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:15:13,684--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:15:13,685--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:15:13,872--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:15:13,874--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:15:14,087--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:15:14,362--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:15:14,756--[test_recharge.py--line:52]--日志等级:INFO--具体内容：用例充值成功执行未通过
2019-09-09 18:15:14,756--[test_recharge.py--line:53]--日志等级:ERROR--具体内容：2000 != Decimal('0.00')
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_recharge.py", line 49, in test_recharge
    self.assertEqual(money,change_money)
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 2000 != Decimal('0.00')
2019-09-09 18:15:14,947--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:15:15,143--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:15:15,355--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:15:15,549--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:15:15,749--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:15:15,750--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,087--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:15:16,088--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,196--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,283--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,343--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,406--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,517--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:15:16,583--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:27:59,646--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:28:10,758--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:28:11,046--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:28:11,046--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:28:11,293--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:28:11,294--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:28:11,580--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:28:11,580--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:28:11,881--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:28:11,882--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:28:12,188--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:28:12,427--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:28:13,214--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值成功执行通过
2019-09-09 18:28:13,406--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:28:13,605--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:28:13,787--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:28:14,023--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:28:14,245--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:28:14,245--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:14,625--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:28:14,626--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:14,748--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:14,859--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:14,932--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:15,068--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:15,444--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:28:15,638--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:31:00,044--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:31:03,118--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:31:03,394--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:31:03,395--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:31:03,625--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:31:03,626--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:31:03,881--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:31:03,882--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:31:04,078--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:31:04,079--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:31:04,300--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:31:04,573--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:31:05,164--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:31:05,892--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:31:06,078--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:31:06,271--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:31:06,456--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:31:06,659--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:31:06,660--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:06,943--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:31:06,944--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:07,100--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:07,163--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:07,257--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:07,316--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:07,393--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:31:07,457--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:32:05,832--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:32:07,828--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:32:08,066--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:32:08,066--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:32:08,350--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:32:08,351--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:32:08,569--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:32:08,570--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:32:08,743--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:32:08,744--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:32:08,953--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:32:09,154--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:32:09,570--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:32:10,090--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:32:10,269--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:32:10,440--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:32:10,649--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:32:10,822--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:32:10,822--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,211--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:32:11,213--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,300--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,358--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,417--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,477--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,539--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:32:11,599--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:34:01,184--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:34:02,572--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:34:02,817--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:34:02,818--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:34:03,014--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:34:03,015--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:34:03,254--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:34:03,255--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:34:03,452--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:34:03,453--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:34:03,650--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:34:03,912--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:34:04,409--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:34:04,994--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值成功：小数执行通过
2019-09-09 18:34:05,210--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:34:05,430--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:34:05,608--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:34:05,802--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:34:05,981--[test_recharge.py--line:57]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:34:05,983--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,376--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:34:06,377--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,503--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,621--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,760--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,845--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,928--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:34:06,993--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:39:19,565--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:39:25,334--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:39:25,551--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:39:25,551--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:39:25,763--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:39:25,764--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:39:26,004--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:39:26,004--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:39:26,196--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:39:26,197--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:39:26,383--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:39:26,606--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:39:27,103--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:39:27,571--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：小数执行通过
2019-09-09 18:39:27,781--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:39:28,005--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:39:28,230--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:39:28,467--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:39:28,690--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:39:28,691--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:28,992--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:39:28,995--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:29,109--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:29,169--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:29,230--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:29,298--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:29,366--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:39:29,636--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:39:30,095--[test_withdraw.py--line:54]--日志等级:INFO--具体内容：用例充值成功：整数执行未通过
2019-09-09 18:39:30,095--[test_withdraw.py--line:55]--日志等级:ERROR--具体内容：2000 != -2000.0
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_withdraw.py", line 51, in test_withdraw
    self.assertEqual(money,float(change_money))
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 2000 != -2000.0
2019-09-09 18:39:30,604--[test_withdraw.py--line:54]--日志等级:INFO--具体内容：用例充值成功：小数执行未通过
2019-09-09 18:39:30,605--[test_withdraw.py--line:55]--日志等级:ERROR--具体内容：500.5 != -500.5
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_withdraw.py", line 51, in test_withdraw
    self.assertEqual(money,float(change_money))
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 500.5 != -500.5
2019-09-09 18:39:30,799--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:39:30,995--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:39:31,207--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:39:31,397--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:39:31,584--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:39:31,585--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:42:48,660--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:42:51,430--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:42:51,648--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:42:51,649--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:42:51,861--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:42:51,862--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:42:52,062--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:42:52,063--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:42:52,253--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:42:52,254--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:42:52,436--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:42:52,635--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:42:53,097--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:42:53,656--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：小数执行通过
2019-09-09 18:42:53,847--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:42:54,026--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:42:54,192--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:42:54,375--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:42:54,544--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:42:54,545--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:54,845--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:42:54,846--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:54,985--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:55,051--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:55,116--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:55,173--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:55,243--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:42:55,618--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:42:56,071--[test_withdraw.py--line:54]--日志等级:INFO--具体内容：用例充值成功：整数执行未通过
2019-09-09 18:42:56,072--[test_withdraw.py--line:55]--日志等级:ERROR--具体内容：2000 != -2000.0
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_withdraw.py", line 51, in test_withdraw
    self.assertEqual(money,float(change_money))
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 2000 != -2000.0
2019-09-09 18:42:56,558--[test_withdraw.py--line:54]--日志等级:INFO--具体内容：用例充值成功：小数执行未通过
2019-09-09 18:42:56,558--[test_withdraw.py--line:55]--日志等级:ERROR--具体内容：500.5 != -500.5
Traceback (most recent call last):
  File "E:\Testsoft\python.project\interface\interface_object03\testcases\test_withdraw.py", line 51, in test_withdraw
    self.assertEqual(money,float(change_money))
  File "E:\Testsoft\lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "E:\Testsoft\lib\unittest\case.py", line 832, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 500.5 != -500.5
2019-09-09 18:42:56,732--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:42:56,907--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:42:57,074--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:42:57,295--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:42:57,518--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:42:57,520--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:44:14,951--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:44:17,747--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:44:17,967--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:44:17,968--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:44:18,214--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:44:18,214--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:44:18,456--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:44:18,457--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:44:18,702--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:44:18,703--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:44:18,883--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:44:19,115--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:44:19,965--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:44:20,511--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：小数执行通过
2019-09-09 18:44:20,733--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:44:20,948--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:44:21,277--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:44:21,550--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:44:21,810--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:44:21,810--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,164--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:44:22,167--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,260--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,320--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,379--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,438--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,518--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:44:22,588--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:45:41,775--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:45:47,104--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:45:47,350--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:45:47,351--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:45:47,641--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:45:47,641--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:45:47,838--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:45:47,839--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:45:48,013--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:45:48,014--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:45:48,207--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:45:48,437--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:45:48,942--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:45:49,346--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：小数执行通过
2019-09-09 18:45:49,516--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:45:49,695--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:45:49,864--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:45:50,047--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:45:50,217--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:45:50,218--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:50,537--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:45:50,539--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:50,661--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:50,717--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:50,775--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:50,843--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:50,909--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:45:51,210--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:45:51,644--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现成功执行通过
2019-09-09 18:45:51,813--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额为空执行通过
2019-09-09 18:45:51,978--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额大于2位小数执行通过
2019-09-09 18:45:52,184--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额为负数执行通过
2019-09-09 18:45:52,405--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额大于50万执行通过
2019-09-09 18:45:52,573--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额格式错误执行通过
2019-09-09 18:45:52,837--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，提现金额大于账户余额执行通过
2019-09-09 18:45:52,839--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
2019-09-09 18:58:50,783--[suite_run.py--line:18]--日志等级:INFO--具体内容：------开始执行测试运行程序------
2019-09-09 18:58:53,780--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:58:54,047--[test_login.py--line:56]--日志等级:DEBUG--具体内容：登录成功用例执行通过
2019-09-09 18:58:54,048--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:58:54,283--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码错误用例执行通过
2019-09-09 18:58:54,284--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:58:54,620--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号错误用例执行通过
2019-09-09 18:58:54,620--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:58:54,899--[test_login.py--line:56]--日志等级:DEBUG--具体内容：密码为空用例执行通过
2019-09-09 18:58:54,899--[test_login.py--line:36]--日志等级:INFO--具体内容：正在请求地址http://test.lemonban.com/futureloan/mvc/api/member/login
2019-09-09 18:58:55,135--[test_login.py--line:56]--日志等级:DEBUG--具体内容：手机号为空用例执行通过
2019-09-09 18:58:55,419--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:58:55,922--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：整数执行通过
2019-09-09 18:58:56,385--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值成功：小数执行通过
2019-09-09 18:58:56,588--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为空执行通过
2019-09-09 18:58:56,769--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于2位小数执行通过
2019-09-09 18:58:56,947--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额为负数执行通过
2019-09-09 18:58:57,173--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额大于50万执行通过
2019-09-09 18:58:57,352--[test_recharge.py--line:59]--日志等级:INFO--具体内容：用例充值失败，金额格式错误执行通过
2019-09-09 18:58:57,353--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:57,736--[test_register.py--line:49]--日志等级:DEBUG--具体内容：用例:注册成功测试不通过
2019-09-09 18:58:57,739--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:57,818--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:57,895--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:57,963--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:58,066--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:58,138--[test_register.py--line:34]--日志等级:INFO--具体内容：请求地址:http://test.lemonban.com/futureloan/mvc/api/member/register
2019-09-09 18:58:58,460--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例正常登录执行通过
2019-09-09 18:58:58,968--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现成功执行通过
2019-09-09 18:58:59,205--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额为空执行通过
2019-09-09 18:58:59,421--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额大于2位小数执行通过
2019-09-09 18:58:59,667--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额为负数执行通过
2019-09-09 18:58:59,963--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额大于50万执行通过
2019-09-09 18:59:00,236--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，金额格式错误执行通过
2019-09-09 18:59:00,522--[test_withdraw.py--line:59]--日志等级:INFO--具体内容：用例提现失败，提现金额大于账户余额执行通过
2019-09-09 18:59:00,525--[suite_run.py--line:41]--日志等级:INFO--具体内容：------用例执行结束------
